src/network/networkd-link.c

```diff
@@ -2820,6 +2834,22 @@ int link_update(Link *link, sd_netlink_message *m) {
                         return r;
         }
 
+        /* find whether link is enslaved */
+        r = sd_netlink_message_read_u32(m, IFLA_MASTER, &master);
+        if (r >= 0) {
+
+                r = sd_netlink_message_enter_container(m, IFLA_PROTINFO);
+                if (r >= 0) {
+                        uint8_t state;
+
+                        r = sd_netlink_message_read_u8(m, IFLA_BRPORT_STATE, &state);
+                        if (r >= 0)
+                                link->bridge_port = true;
```

Comment:
`bridge_port` is set here, but it is never unset. Shouldn't it be set to false whenever IFLA_BRPORT_STATE is not found?